### PR TITLE
run make upgrade to update jinja2

### PR DESCRIPTION
**Pull request by DawoudSheraz**
**Description:** 

This PR updates the jinja2 version. Please see PROD-343 for more information.

**JIRA:**

[PROD-343](https://openedx.atlassian.net/browse/PROD-343)

**Dependencies:**

List dependencies on other outstanding PRs, issues, etc.

**Merge deadline:**

List merge deadline (if any)

**Installation instructions:**

List any non-trivial installation
instructions.

**Testing instructions:**

1. Open page A
2. Do thing B
3. Expect C to happen
4. If D happened instead - check failed.

**Reviewers:**
- [ ] @edx/arch-review
- [ ] tag reviewer

**Merge checklist:**
- [ ] All reviewers approved
- [x] CI build is green
- [x] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed
- [ ] PR author is listed in AUTHORS

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPi after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

**Author concerns:**

List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about 
migrations, etc.

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -4,8 +4,8 @@
 #
 #    make upgrade
 #
-django-waffle==0.14.0
-django==1.11.15
-newrelic==4.2.0.100
+django-waffle==0.16.0
+django==1.11.20
```
Comment (robrap) [inline requirements/base.txt]: According to an email from Feanil, 1.11.21 was a security release.  I'm not sure why that wasn't picked up.  If you run again do you get it?

Comment (DawoudSheraz) [reply requirements/base.txt]: The version release is due on June 3rd, 2019.

**Comment by DawoudSheraz** [comment]
@robrap I created this test PR on the edx-platform against this branch. 
https://github.com/edx/edx-platform/pull/20726

**Comment by DawoudSheraz** [comment]
@robrap Sorry to mention again, but I have updated this PR. There were some missing requirements constraints and a few quality fixes that were required for Travis builds to complete successfully. If you can have another look, and if it looks ok, can you merge it? I don't have merge access.

Thank You

**Pull request merged**